src/components/common/ColonyActions/CountDownTimer/CountDownTimer.tsx

```diff
@@ -36,7 +36,7 @@ interface Props {
   motionState: MotionState;
   refetchMotionState: RefetchMotionState;
   motionId: string;
-  motionStakes: MotionData['motionStakes'];
```

Comment:
Related to my previous comment, by keeping these dependant on `MotionData` fragment we were sure we're working with the data available from the query

Reply:
This was on my suggestion based on the general pattern we're following of not accessing nested types.